index.html

```diff
@@ -0,0 +1,251 @@
+                    />
+                  </svg>
+                </a>
+              </li>
+
+              <li>
+                <a href="#" class="display-on-desk">English</a>
+              </li>
+              <li>
+                <a href="#" class="display-on-desk">MyPage</a>
+              </li>
+              <li>
+                <a href="#" class="display-on-desk">Logout</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+
+        <!-- second nav bar -->
+        <nav class="header-nav-two">
+          <a href="./index.html"
+            ><img src="./assets/summit-logo.png" alt="logo" class="logo"
+          /></a>
+
+          <div class="nav-two-container">
+            <ul class="nav-list">
+              <li class="nav-item"><a href="./html/about.html">About</a></li>
+              <li class="nav-item"><a href="#program">Program</a></li>
+              <li class="nav-item"><a href="#join">Join</a></li>
+              <li class="nav-item"><a href="#sponsor">Sponsor</a></li>
+              <li class="nav-item"><a href="#">News</a></li>
+            </ul>
+          </div>
+
+          <button class="nav-btn" type="button">Register</button>
+        </nav>
+      </div>
+      <!-- humberger menu -->
+      <div class="header-humberger-menu">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </header>
+
+    <!-- Main content -->
+    <main class="main">
+      <!-- introduction seection -->
+      <section class="main-intro">
+        <h1 class="main-intro-title">Mental Health Global Summit 2022</h1>
+        <div class="main-intro-description-cont">
+          <p class="main-intro-description">
+            Join people from over 100 countries in securing a healthy and a
+            happy life.
+          </p>
+        </div>
+        <div class="main-intro-dates">
+          <span class="main-intro-upper">2022.10.20(THU) ~ 21(FRI) </span>
+          <br />
+          <p>@ Madison Square Garden | NewYork - USA</p>
+        </div>
+      </section>
+
+      <!-- main program section -->
+      <section id="program" class="main-program">
+        <h2 class="main-program-title">Main Program</h2>
+        <div class="main-program-cards">
+          <!-- card -->
+          <div class="card">
+            <img
+              src="./assets/public-speaking.svg"
+              alt="public speaking"
+              class="card-icon"
+            />
+            <h3 class="card-title">Lecture</h3>
+            <p class="card-description">
+              Listen to experts in the field of mental health about mental
+              health and how to curb it
+            </p>
+          </div>
```

Comment:
- [ ] Using semantic HTML, it would be more appropriate to have `<article class="card">` in place of the `<div>` tag here because they are actually articles 👍